packages/commands/src/components/command-menu.js

```diff
@@ -208,6 +206,16 @@ export function CommandMenu() {
 		} );
 	}, [ registerShortcut ] );
 
+	// Temporary fix for the suggestions Listbox labeling.
+	// See https://github.com/pacocoursey/cmdk/issues/196
+	useEffect( () => {
+		commandListRef.current?.removeAttribute( 'aria-labelledby' );
+		commandListRef.current?.setAttribute(
+			'aria-label',
+			__( 'Command suggestions' )
+		);
+	}, [ commandListRef.current ] );
```

Comment:
Is this really necessary, I'd prefer if we avoid this hack personally. We're updating a DOM node controller by React using an imperative API, this can break in different ways. (For example, if the component is re-rendered, these things could get reset).

Reply:
@youknowriad Yes I know it's hacky 😞 I reported the root problem upstream at https://github.com/pacocoursey/cmdk/issues/196. If you could maybe help getting the `cmdk` team release a fix soon we could entirely avoid this hack